src/lib.rs

```diff
@@ -314,6 +329,21 @@ impl CANSocket {
         Ok(())
     }
 
+    /// Control Flexible Data Rate
+    #[cfg(feature = "can_fd")]
+    pub fn set_fd_frames(&self, fd_frames_enable: bool) -> io::Result<()> {
+        let fd_frames_enable = fd_frames_enable as c_int;
+        let opt_ptr = &fd_frames_enable as *const c_int;
+        let rv = unsafe {
+            setsockopt(self.fd, SOL_CAN_RAW, CAN_RAW_FD_FRAMES, opt_ptr as *const c_void, size_of::<c_int>() as u32)
+        };
+
+        if rv != 0 {
```

Comment:
I think just consistency maybe `== -1` would be better ?

Reply:
Ok